src/test/java/se/sundsvall/oepintegrator/api/WebMessageResourceTest.java

```diff
@@ -0,0 +1,84 @@
+package se.sundsvall.oepintegrator.api;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+import static org.springframework.http.MediaType.ALL;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.http.MediaType.MULTIPART_FORM_DATA;
+import static org.springframework.http.MediaType.TEXT_PLAIN;
+import static se.sundsvall.oepintegrator.utility.Constants.REFERENCE_FLOW_INSTANCE_ID;
+
+import callback.AddMessageResponse;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.client.MultipartBodyBuilder;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.bean.override.mockito.MockitoBean;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import org.springframework.web.reactive.function.BodyInserters;
+import se.sundsvall.oepintegrator.Application;
+import se.sundsvall.oepintegrator.api.model.webmessage.ExternalReference;
+import se.sundsvall.oepintegrator.api.model.webmessage.Sender;
+import se.sundsvall.oepintegrator.api.model.webmessage.WebMessageRequest;
+import se.sundsvall.oepintegrator.integration.db.model.enums.InstanceType;
+import se.sundsvall.oepintegrator.service.WebMessageService;
+
+@SpringBootTest(classes = Application.class, webEnvironment = RANDOM_PORT)
+@ActiveProfiles("junit")
+class WebMessageResourceTest {
```

Comment:
This is the happy case test, Shouldn't we have WebMessageResourceFailuresTest as well, in order to test the API-constraints?